.env.example

```diff
@@ -79,6 +79,11 @@ DEFAULT_PAGING=500
 # disable the help video feature.
 HELP_VIDEO_URL=https://www.youtube.com/embed/ScMzIvxBSi4?rel=0&enablejsapi=1
 
+# Whether to show help video on first landing
+# 1: Do not show
+# 0: show
+SHOW_HELP_VIDEO_FIRST_LANDING=1
+
```

Comment:
I think the logic here is kind of backwards. Would it make sense to use `1` as enable and `0` as disable? :grinning: